go/auth/credential_authority.go

```diff
@@ -226,9 +227,13 @@ func (v *CredentialAuthority) runWithCancel(body func(ctx context.Context) error
 // pollLoop() keeps running until the CA is shut down via Shutdown(). It calls Poll()
 // on the UserKeyAPIer once per iteration.
 func (v *CredentialAuthority) pollLoop() {
-	var err error
-	for err != ErrShutdown {
-		err = v.pollOnce()
+	for {
+		if err := v.pollOnce(); err != nil {
```

Comment:
Yeah, could add an additional wait I guess of a like a second if we think that is better.